src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/LegacyBqlField/Sources/LegacyBqlFieldBad.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using PX.Data;
+
+namespace PX.Analyzers.Test.Sources
+{
+	public class BadDac : IBqlTable
+	{
+		public abstract class legacyField : IBqlField { }
```

Comment:
@Derrekam please add more fields to the test. It will be good to test field of byte[] array type, field of primitive type like int, field of primitive type which is written with .Net name like Int32, maybe some exotic (uint16[]) or not supported fields.

Reply:
done